README.md

```diff
@@ -27,3 +27,46 @@ Inside your module you can now watch for template changes :
 npm run watch  
 
 ```
+
+## Usage Grunt
+Add the dependency to your project using npm:
+
+    npm install --save-dev fh-wfm-template-build
+
+Load the plugin by adding it to your Gruntfile.js:
+
+    grunt.loadNpmTasks('fh-wfm-template-build');
+
+Add a configuration to your Gruntfile.js :
+
+    grunt.initConfig({
+      wfm_build: {
+        options: {
+          module: "wfm.<module-name>.directives",
+        }
+      },
+      wfm_watch: {
+        options: {
+          module: "wfm.<module-name>.directives",
+        }
+      },
```

Comment:
I'll take a look at this and see what can be done. 

I've not used the commands in question but since there are two different ones, `build` and `watch` do we want to keep two task for this or perhaps one multitask using something like `grunt wfmTemplate:build` and `grunt wfmTemplate:watch` but share the same configuration for them both? 

With the configuration that you have one would have to update the `watch` property which is the reason for my asking. Thanks